ansible/tasks/logging/install_fluentd.yaml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Create temporary directory on windows host
+  win_tempfile:
+    state: directory
+  register: tempdir_win
+- name: Fetch ruby devkit exe
+  win_get_url:
+    url: "{{ ruby_fetch_url }}"
+    dest: "{{ tempdir_win.path }}\\ruby_devkit.exe"
+- name: Run ruby devkit exe
+  win_package:
+    path: "{{ tempdir_win.path }}\\ruby_devkit.exe"
+    creates_path: "C:\\Ruby26-x64\\bin\\ruby.exe"
+    state: present
+    arguments:
+    - /VERYSILENT
+    - /SUPPRESSMSGBOXES
+- name: Install fluentd
+  win_shell: gem install fluentd
```

Comment:
Where do we specify the version of fluentd that we want?

Reply:
Right now we do not, so it is fetching the latest stable version of fluentd (currently 1.7.3). We can pin the version here - I will test and include the version mentioned on [origin aggregated logging](https://github.com/openshift/origin-aggregated-logging/commit/8ab73fbbfc5172d69ec81fb63dabc77196415ee6)

Reply:
Right now this installs the latest version. The new commit should give the option of providing the version of fluentd installation